make_zip_on_aws.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function
+
+import time
+import os
+import sys
+
+from boto.ec2 import connect_to_region
+from paramiko.client import SSHClient, AutoAddPolicy
+from paramiko.sftp_client import SFTPClient
+from paramiko.ssh_exception import NoValidConnectionsError
+
+# http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html
+AWS_REGION = "us-west-2"
+AMI_ID = "ami-f0091d91"
+KEY_PAIR = "loads"
+SECURITY_GROUP = "loads"
+INSTANCE_TYPE = "t2.micro"
+INSTANCE_NAME = "amo2kinto-lambda-zip-builder"
+INSTANCE_PROJECT = "amo2kinto-lambda"
+
+# 1. Connect on the AWS region
+print("Connecting to %s" % AWS_REGION)
+conn = connect_to_region(AWS_REGION, is_secure=True)
+
+# 2. Create a Amazon Lambda AMI EC2 instance
+print("Starting an new instance of %s" % AMI_ID)
+reservations = conn.run_instances(AMI_ID,
+                                  min_count=1, max_count=1,
+                                  key_name=KEY_PAIR,
+                                  security_groups=[SECURITY_GROUP],
+                                  instance_type=INSTANCE_TYPE)
+
+instance = reservations.instances[0]
+
+# 3. Tag the instance
+conn.create_tags([instance.id], {
+    "Name": INSTANCE_NAME,
+    "Projects": INSTANCE_PROJECT,
+})
+print("Instance Name:", "amo2kinto-lambda-zip-builder")
+
+# 4. Wait for running
+while instance.state != "running":
+    print("\rInstance state: %s" % instance.state, end="")
+    sys.stdout.flush()
+    time.sleep(10)
```

Comment:
it would be safer to add a time.sleep() under the while loop because I've experienced some race conditions in the past where the state was just flipped to running and the box not listening to ssh *yet*